tests/slsa_analyzer/test_analyzer.py

```diff
@@ -42,49 +42,6 @@ def test_resolve_local_path(self) -> None:
         assert Analyzer._resolve_local_path(self.PARENT_DIR, "././././") == self.PARENT_DIR
 
 
-@pytest.mark.parametrize(
```

Comment:
I saw that you moved the `to_repo_path` function from `slsa_analyzer/analyzer.py` to `repo_finder/repo_finder.py`.
I assume that means this test should also be moved and not removed? I don't think I find this test being moved anywhere so I assume it is being removed.

Reply:
Good catch. I am fairly sure I moved it, but now I can't find it. I'll add it back.